numpy/lib/tests/test_io.py

```diff
@@ -1823,6 +1828,123 @@ def test_auto_dtype_largeint(self):
         assert_equal(test['f1'], 17179869184)
         assert_equal(test['f2'], 1024)
 
+
+class TestPathUsage(TestCase):
+    @contextlib.contextmanager
```

Comment:
There is another PR for adding a context manager for temporary files, see #6545 and comments there.